src/category_theory/limits/shapes/constructions/finite_limits.lean

```diff
@@ -44,8 +48,9 @@ let t : pi_obj ⟶ pi_hom :=
   pi.lift (λ f : (Σ p : J × J, p.1 ⟶ p.2), pi.π F.obj f.1.2) in
 parallel_pair s t
 
-@[simp] def fin_equalizer_diagram.cones_hom [has_finite_products.{v} C] (F : J ⥤ C) :
-  (fin_equalizer_diagram F).cones ⟶ F.cones :=
+/-- See has_limit_of_has_products_of_has_equalizers.cones_hom. -/
```

Comment:
I'm not sure if this is a very valuable docstring... it's better than nothing... but we can't follow such a link by clicking on it in the pop-up window.

Reply:
Solved the problem by completely deleting all the repeated code, and just slightly generalising the arguments in the construction so it handles both cases at once... This is much much better now.